README.md

```diff
@@ -82,6 +80,108 @@ You can also run this as a scraper on [Morph](https://morph.io).
 
 To get started [see the documentation](https://morph.io/documentation)
 
+## Contributing
+
+## Adding new runners to Jacaranda
+
+Jacaranda has a very simple model for adding new runners.
+
+Runners pull information from (sometimes multiple) sources, and posts a message into Slack.
+
+To add a new runner, open up `scraper.rb` and define the following class:
+
+``` ruby
+module Jacaranda
+  # A new runner for my new service
+  class MyNewRunner < Runner
+    class << self
+      def build
+        [
+          'My text here.'
+        ]
+      end
+    end
+  end
+end
+```
+
+Then run the scraper:
+
+``` bash
+MORPH_LIVE_MODE=false bundle exec ruby scraper.rb --runners MyNewRunner
```

Comment:
This implies you need the `--runners` option to run your runner—which is makes it unclear how this could run on morph.io. I think a sentence explaining that would be helpful.